README.md

```diff
@@ -0,0 +1,96 @@
+# UI-KIOSK
+
+![GitHub package.json version](https://img.shields.io/github/package-json/v/UniversityOfGdanskProjects/UI-KIOSK?style=for-the-badge)
+![GitHub top language](https://img.shields.io/github/languages/top/UniversityOfGdanskProjects/UI-KIOSK?logo=typescript&style=for-the-badge)
+![Netlify](https://img.shields.io/netlify/9aef6243-0742-42b4-9636-049204e0bbed?logo=netlify&style=for-the-badge)
+![GitHub package.json dependency version (prod)](https://img.shields.io/github/package-json/dependency-version/UniversityOfGdanskProjects/UI-KIOSK/react?color=green&logo=react&style=for-the-badge)
+![GitHub package.json dependency version (dev dep on branch)](https://img.shields.io/github/package-json/dependency-version/UniversityOfGdanskProjects/UI-KIOSK/dev/typescript/master?color=darkblue&logo=typescript&style=for-the-badge)
+![GitHub package.json dependency version (dev dep on branch)](https://img.shields.io/github/package-json/dependency-version/UniversityOfGdanskProjects/UI-KIOSK/dev/vite/master?color=brown&logo=vite&style=for-the-badge)
+
+## Description
+
+UI-KIOSK is a front-end project made for University of Gdańsk. It will be used in the information kiosk located outside of MFI building.
+
+### Technologies
+
+UI-KIOSK uses:
+
+- [Typescript](https://www.typescriptlang.org/)
+- [React](https://reactjs.org/)
+
+### Bundler
+
+This project's bundler is [Vite](https://vitejs.dev/).
```

Comment:
Maybe change this sentence to something like: `this project is bundled by vite`? I think it would sound a bit better

Reply:
resolved